### PR TITLE
Translate into da

### PR DESCRIPTION
**Pull request by haarbo**
Translated da.po into danish

### CODE REVIEW

File: share/da.po [hunk 1]
```diff
@@ -4,139 +4,139 @@ msgstr ""
...
-msgstr "Print version information and exit."
+msgstr "Udskriv version og afbryd."
 
 msgid "The minimum severity level to display"
-msgstr "The minimum severity level to display"
+msgstr "Det mindste alvorlighedsniveau som skal udskrives."
 
 msgid "The locale to use for messages translation."
-msgstr "The locale to use for messages translation."
+msgstr "Landestandarden som skal bruges ved udskrift."
```
Comment (matsduf) [inline share/da.po]: The term "locale" is special and probably not translatable (I notice that the Swedish translation is strange too). I suggest something like (please correct my Danish):
```
"Den 'locale' som skal bruges ved oversættelse af beskeder."
```

**Comment by matsduf** [comment]
This PR will resolve #61.

**Comment by sandoche2k** [comment]
@matsduf make the changes and merge

**Comment by matsduf** [comment]
@haarbo, please review my suggestion of change in your PR.

**Comment by haarbo** [comment]
The suggestion is hereby approved.

Tried to do a new pull request – not sure if I succeded.

/Niels

**Pull request merged**